modules/pam_tty_audit/pam_tty_audit.c

```diff
@@ -351,6 +351,15 @@ pam_sm_open_session (pam_handle_t *pamh, int flags, int argc, const char **argv)
 
   fd = nl_open ();
   if (fd == -1
+      && errno == EPROTONOSUPPORT)
+    {
+      pam_syslog (pamh, LOG_WARNING, "warning: unable to open communication "
```

Comment:
Could you please change the wording of the log message to something like: `unable to open audit socket, audit not supported; tty_audit skipped`

Reply:
Done